app/models/child.rb

```diff
@@ -98,4 +98,8 @@ def to_csv
       archived
     ]
   end
+
+  def export_json
+    { id: id, first_name: first_name, last_name: last_name, date_of_birth: date_of_birth, gender: gender, child_lives_with: child_lives_with, race: race, agency_child_id: agency_child_id, health_insurance: health_insurance, comments: comments, created_at: created_at, updated_at: updated_at, family_id: family_id, item_needed_diaperid: item_needed_diaperid, active: active, archived: archived }
```

Comment:
If these are all the attributes of the `Child`, you could shorten to use `#attributes`. When you do `#attributes` it will output all the attributes as a hash :).

You also might want to add `.to_json` in the end, since the method name `export_json` implies a JSON is supposed to be returned.